packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -1,8 +1,103 @@
 import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
 import { StorageBrowserElements } from '../../context/elements';
 
-const { Table, TableBody, TableData, TableHead, TableHeader, TableRow } =
-  StorageBrowserElements;
+const {
+  Table: BaseTable,
+  TableBody: BaseTableBody,
+  TableData: BaseTableData,
+  TableHead: BaseTableHead,
+  TableHeader: BaseTableHeader,
+  TableRow: BaseTableRow,
+  Button,
+  Icon,
+} = StorageBrowserElements;
+
+const BLOCK_NAME = 'table';
+
+const Table = withBaseElementProps(BaseTable, {
+  className: `${BLOCK_NAME}`,
+});
+
+const TableBody = withBaseElementProps(BaseTableBody, {
+  className: `${BLOCK_NAME}__body`,
+});
+
+const TableHead = withBaseElementProps(BaseTableHead, {
+  className: `${BLOCK_NAME}__head`,
+});
+
+const TableHeader = withBaseElementProps(BaseTableHeader, {
+  className: `${BLOCK_NAME}__header`,
+});
+
+const TableHeaderButton = withBaseElementProps(Button, {
+  className: `${BLOCK_NAME}__header-button`,
+});
+
+const TableData = withBaseElementProps(BaseTableData, {
+  className: `${BLOCK_NAME}__data`,
+});
+
+const TableDataButton = withBaseElementProps(Button, {
+  className: `${BLOCK_NAME}__data-button`,
+});
+
+const TableRow = withBaseElementProps(BaseTableRow, {
+  className: `${BLOCK_NAME}__row`,
+});
+
+const iconAttributes = {
+  'aria-hidden': true,
+  width: '24',
+  height: '24',
+  viewBox: '0 -960 960 960',
+  fill: 'none',
+  xmlns: 'http://www.w3.org/2000/svg',
+};
+
+const SortIndeterminateIcon = withBaseElementProps(Icon, {
+  children: (
+    <path
+      d="M480-360 280-560h400L480-360Z"
+      fill="none"
+      stroke="currentColor"
+      strokeWidth="1"
+      vectorEffect="non-scaling-stroke"
```

Comment:
Added these attributes to create a similar triangle from the figma file:

![image](https://github.com/user-attachments/assets/7e4fcad0-df10-4a46-b7fa-5167a4fb124f)

Reply:
Use https://fonts.google.com/icons?selected=Material+Symbols+Outlined:check_indeterminate_small:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=indeter&icon.size=24&icon.color=%23e8eaed for the indeterminate icon